ccm_web/client/src/components/BulkSectionCreateFileExample.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
```

Comment:
```suggestion

```

Reply:
This seems to be required to me, even though it's not directly referenced.  If I remove it it says 'React' must be in scope when using JSX.  Is there something better to do than leave it in?

Reply:
@ssciolla does it make sense to have the rule ` "react/react-in-jsx-scope": "off",` in the `.eslintrc.js`?

Reply:
Um, I may not know enough about this, but this seems like a separate issue. I thought React has to be in scope to use JSX, so I've just become accustomed to ignoring that it's not explicitly referenced. In VSCode for me, it's usually just grayed out, not a warning or anything. My vote would be to keep at as is for now until we understand better.

Reply:
Yeah.. for me too as well grayed out. Ok will leave it as it is now. While doing more of Typescript we can build our understanding. I will resolve it now